content/version/1/4/code-of-conduct.it.md

```diff
@@ -0,0 +1,83 @@
++++
+version = "1.4"
+aliases = ["/version/1/4/it"]
++++
+
+# Codice di Comportamento dei Collaboratori
+
+## Il nostro impegno
+
+Nell'interesse di promuovere un ambiente aperto ed accogliente, noi come
+collaboratori e responsabili ci impegnamo a rendere la partecipazione al nostro 
```

Comment:
partecipazione del nostro

Reply:
I'm sorry @rikace , the comment is the correction or error?

Reply:
is the correction. It should be 
`collaboratori e responsabili ci impegnamo a rendere la partecipazione del nostro`

Reply:
Ok. I do not agree with you on this point: I think it is a "complemento di termine" so it is better   "collaboratori e responsabili ci impegnamo a rendere la partecipazione al nostro".

Reply:
you are right, however, 
"complemento di termine" ... shouldn't be "complemento del termine" ??

I don't see this part in the document

Reply:
Hi @rikace, sorry i missed this. The "complemento di termine" is https://it.wikipedia.org/wiki/Complemento_di_termine so it is better "collaboratori e responsabili ci impegnamo a rendere la partecipazione _**al nostro progetto e alla nostra comunità**_". I think.